### PR TITLE
backend services pages

### PR DESCRIPTION
**Pull request by Healthedata1**
image

### CODE REVIEW

File: input/images-source/overview-backend-services.plantuml [hunk 1]
```diff
@@ -3,16 +3,16 @@ participant App as "Backend Service"
 participant "FHIR authorization server" as EHR
 participant "FHIR resource server" as FHIR
 
-opt  Precondition: Client Registration
+opt  1. Precondition: Client Registration
```
Comment (jmandel) [inline input/images-source/overview-backend-services.plantuml:6]: These numbers don't line up vertically and don't make the diagram clearer for me. I'm hesitant to incorporate them here, especially if we're not going to incorporate them into the section titles below.

**Review by jmandel — CHANGES_REQUESTED** [review]

File: input/images-source/overview-backend-services.plantuml (hunk 1 above)
Comment (Healthedata1) [reply input/images-source/overview-backend-services.plantuml:6]: The numbered list should relate to the sequence diagram .. alternate is to renumber the list to a sequentially numbered sequence diagram.

Comment (jmandel) [reply input/images-source/overview-backend-services.plantuml:6]: We can resolve this at https://github.com/HL7/smart-app-launch/pull/381 and apply our here

Comment (Healthedata1) [reply input/images-source/overview-backend-services.plantuml:6]: OK agree that the diagram suffers.

**Pull request merged**